src/renderer/src/routes/pages/Send/TokenBalanceList.tsx

```diff
@@ -0,0 +1,78 @@
+import { Fragment } from 'react'
+import { useTranslation } from 'react-i18next'
+import { BlockchainIcon } from '@renderer/components/BlockchainIcon'
+import { BalanceHelper } from '@renderer/helpers/BalanceHelper'
+import { StringHelper } from '@renderer/helpers/StringHelper'
+import { StyleHelper } from '@renderer/helpers/StyleHelper'
+import { useAccountsSelector } from '@renderer/hooks/useAccountSelector'
+import { useBalancesAndExchange } from '@renderer/hooks/useBalancesAndExchange'
+
+export const TokenBalanceList = () => {
+  const { t } = useTranslation('pages', { keyPrefix: 'send' })
+  const { accounts } = useAccountsSelector()
+  const balanceExchange = useBalancesAndExchange(accounts)
+  let rowCount = 0
+
+  return (
+    <section className="overflow-y-auto flex flex-col basis-0 flex-grow">
+      <table className="text-left my-2">
+        <thead>
+          <tr className="text-gray-100 text-opacity-60">
+            <th className="w-1/3 pl-4 py-2">{t('token')}</th>
+            <th className="w-1/3">{t('quantity')}</th>
+            <th className="w-1/3">{t('price')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {balanceExchange.balance.data.map(balance => {
+            return (
+              <Fragment key={balance.address}>
+                {balance.tokensBalances.map((tokenBalance, innerIndex) => {
+                  if (tokenBalance.amountNumber > 0) {
+                    const isGrayBackground = rowCount % 2 !== 0
+                    rowCount++
```

Comment:
Why you need it ?

Reply:
Because I need alternating colors between the rows. Since there are 2 foreach loops and there are cases where a line is skipped (if it's == 0), I need this row count for the correct color alternation.